inference-engine/src/transformations/src/transformations/convert_depth_to_space.cpp

```diff
@@ -14,12 +14,17 @@ void ngraph::pass::ConvertDepthToSpace::convert_depth_to_space() {
     auto input0 = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 1, 1});
     auto dts_node = std::make_shared<ngraph::opset1::DepthToSpace>(input0, ngraph::op::DepthToSpace::DepthToSpaceMode::DEPTH_FIRST);
 
-    ngraph::graph_rewrite_callback callback = [](pattern::Matcher& m) {
+    ngraph::graph_rewrite_callback callback = [this](pattern::Matcher& m) {
         auto dts_node = std::dynamic_pointer_cast<ngraph::opset1::DepthToSpace> (m.get_match_root());
         if (!dts_node) {
             return false;
         }
 
+        // disable the transformation for CPU plug-in
```

Comment:
We do not specify plugins specific comments in transformations. Every plugin can use callback and enable/disable this transformation.

Reply:
It will be fixed, but how can we quickly get information about the status (enable/disable)  of transformations for each plugin? Do we have a centralized place with this info or we should look at the corresponding file for each plugin where callbacks are called?

Reply:
Transformations do not know anything about their usage in plugins and there is no centralized place. You can find callback in each plugin but if you want to specify which transformations it disables, please place your comment to plugins directly. In case of CPU: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp#L84